dicomsort.py

```diff
@@ -307,7 +307,7 @@ def usage():
     print("\n <patterns...> is a string defining the output file and directory")
     print("names based on the dicom tags in the file.")
     print("\n Examples:")
-    print("\n  dicomsort data sorted/%PatientName/%StudyDate/%SeriesDescription-%InstanceUID.dcm")
+    print("\n  dicomsort data sorted/%PatientName/%StudyDate/%SeriesDescription-%SOPInstanceUID.dcm")
```

Comment:
Sorry, I just realized you perhaps meant `InstanceNumber` here? Just from looking at the comment in the next line.

Reply:
Good catch.  Yes, should say InstanceNumber not UID.

Fixed in: https://github.com/pieper/dicomsort/commit/68a5935127367df82e3338957f064456254c0bde